packages/cadl-ranch-specs/http/payload/multipart/mockapi.ts

```diff
@@ -164,3 +164,11 @@ Scenarios.Payload_MultiPart_FormData_checkFileNameAndContentType = passOnSuccess
 Scenarios.Payload_MultiPart_FormData_anonymousModel = passOnSuccess(
   createMockApis("anonymous-model", [checkProfileImage]),
 );
+
+Scenarios.Payload_MultiPart_FormData_checkFileNameAndContentTypeWithHttpPart = passOnSuccess(
```

Comment:
That also be better as 2 separate test and ideally more
Required file name 
Optional filename 
Required content type
Optional content type

Reply:
Specific content type

Reply:
my point for the 1st one is that you have `complex` as a test which is bad(it is not representative of any feature if you look at the dashboard), this should be split into the individual feature it is testing

Reply:
I see. But I think it is also reasonable for complex scenarios that combine all single scenarios since sometimes `1+1` is much much more complicated than `1`.
I will add separate cases for  `Required file name/Required content type/Specific content type`

Reply:
I think it’s fair to have one multiple scenario to ensure multiple parts work together as well but it shouldn’t be used as the catch all

Reply:
Updated: 
- fileWithHttpPartSpecificContentType for specific content type
- fileWithHttpPartRequiredContentType for required content type
- fileWithHttpPartOptionalContentType for optional content type
- each of up cases cover required filename
- complexWithHttpPart combines up scenarios